js/admin.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+(function(ctx) {
+
+  let template = Handlebars.compile($('#project-template').html());
+
+  function loggedIn() {
+    return localStorage.access_token && parseInt(localStorage.token_expiry) > Date.now();
+  }
+
+  function initLoginForm() {
+    $('.form-login').on('submit', function(e) {
+      $('.form-login').attr('disabled', true);
+      e.preventDefault();
+      let data = $(this).serialize();
+      $.post('http://api.trentonkress.com/api/login', data, function(res) {
+        if (res) {
```

Comment:
are these if(res) and if(res.success) checks needed? isn't this already the success callback? you should be able to get around this if you put the success callback in a .done()